src/lib.rs

```diff
@@ -122,4 +122,4 @@
 pub mod bms;
 pub mod bmson;
 
-pub use bms::{command, lex, parse};
+pub use bms::{ast, command, lex, parse, parse_bms, prelude};
```

Comment:
Why did you re-export `parse_bms` function here? I think it should be in `prelude` rather than the crate root.